python/tests/unit/arcticdb/version_store/test_parallel.py

```diff
@@ -18,11 +18,41 @@
     random_floats,
     random_dates,
 )
+from arcticdb_ext.storage import KeyType
+
+
+def test_remove_incomplete(lmdb_version_store):
```

Comment:
What happens if you remove incomplete data for a symbol that doesn't exist, or that doesn't have any incomplete data?

Reply:
Added to the test covering these cases (the answer is nothing to both)